bootstrap/pkg/kfapp/kustomize/kustomize.go

```diff
@@ -499,29 +386,6 @@ func (kustomize *kustomize) Generate(resources kftypesv3.ResourceEnum) error {
 // Init is called from 'kfctl init ...' and creates a <deployment> directory with an app.yaml file that
 // holds deployment information like components, parameters
 func (kustomize *kustomize) Init(resources kftypesv3.ResourceEnum) error {
```

Comment:
seems like this becomes a placeholder?

Reply:
I think Init is no longer needed but its probably part of the interface so deleting it might be part of a bigger cleanup.

But we could potentially add a TODO that this is no longer needed.